chemprop/nn/predictors.py

```diff
@@ -203,6 +204,30 @@ def forward(self, Z: Tensor) -> Tensor:
     train_step = forward
 
 
+@PredictorRegistry.register("regression-quantile")
+class QuantileFFN(RegressionFFN):
+    n_targets = 2
+    _T_default_criterion = QuantileLoss
+
+    def forward(self, Z: Tensor) -> Tensor:
+        Y = super().forward(Z)
+        lower_bound, upper_bound = torch.chunk(Y, self.n_targets, 1)
+
+        lower_bound = self.output_transform(lower_bound)
+        upper_bound = self.output_transform(upper_bound)
+
+        mean = (lower_bound + upper_bound) / 2
+        interval = upper_bound - lower_bound
+
+        return torch.cat((mean, interval), 1)
+
+    def train_step(self, Z: Tensor) -> Tensor:
+        Y = self.forward(Z)
+        mean, interval = torch.chunk(Y, self.n_targets, 1)
+
+        return torch.cat((mean, interval), 1)
```

Comment:
I'm confused why you chunk the mean and interval and then immediately cat them again.

Reply:
Good catch! I will fix it.